xml/admin_authentication.xml

```diff
@@ -92,12 +92,13 @@
      generated LDAP certificate to your local trusted certificate storage. On
      the &admin_node;, run:
     </para>
-    <screen>&prompt.root;<command>docker</command> exec -it $(docker ps | grep openldap | awk '{print $1}') cat /etc/openldap/pki/ca.crt > ~/ca.pem</screen>
-    <screen>&prompt.root;<command>scp</command> ~/ca.pem <replaceable>root@workstation</replaceable>:/usr/share/pki/trust/anchors/ca-caasp.crt.pem</screen>
+    <screen>&prompt.root;<command>docker exec -it $(docker ps | grep openldap | \
```

Comment:
This line already had the optimal length. I double-checked it works in all output formats.
This command – or very similar ones – are used multiple times in this chapter and I made sure they are all consistent, including the line-break.